department-of-veterans-affairs/caseflow
# Refactor to fetch child hearings by Caseflow hearing day  id both for VACOLS and Caseflow hearings.

Resolves #8990 

### Description
Refactor to fetch child hearings by Caseflow hearing day  id both for VACOLS and Caseflow hearings.

### Acceptance Criteria

### Testing Plan
1. Schedule a legacy appeal for a hearing in Assign Hearings
2. Confirm veteran shows in the list of Scheduled Veterans once you return to Assign Hearings
3. Confirm hearing shows up in the appropriate daily docket.
